frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_statistics_modal.tsx

```diff
@@ -0,0 +1,173 @@
+import { getSegmentBoundingBoxes, getSegmentVolumes } from "admin/admin_rest_api";
+import { Modal, Spin, Table } from "antd";
+import saveAs from "file-saver";
+import { formatNumberToUnit, formatNumberToVolume } from "libs/format_utils";
+import { useFetch } from "libs/react_helpers";
+import { Unicode, Vector3 } from "oxalis/constants";
+import { getResolutionInfo } from "oxalis/model/accessors/dataset_accessor";
+import { Segment } from "oxalis/store";
+import React from "react";
+import { TreeNode } from "./segments_view_helper";
+import { api } from "oxalis/singletons";
+
+const SEGMENT_STATISTICS_CSV_HEADER =
+  "segmendId,segmentName,groupId,groupName,volumeInVoxel,volumeInNm3,boundingBoxTopLeftPositionX,boundingBoxTopLeftPositionY,boundingBoxTopLeftPositionZ,boundingBoxSizeX,boundingBoxSizeY,boundingBoxSizeZ";
+
+const { ThinSpace } = Unicode;
+
+type Props = {
+  onCancel: (...args: Array<any>) => any;
+  isOpen: boolean;
+  tracingId: any;
+  tracingStoreUrl: any;
+  visibleSegmentationLayer: any;
+  relevantSegments: Segment[];
+  parentGroup: number;
+  groupTree: TreeNode[];
+};
+
+type SegmentInfo = {
+  segmentId: number;
+  segmentName: string;
+  groupId: number | undefined | null;
+  groupName: string;
+  volumeInNm3: number;
+  formattedSize: string;
+  volumeInVoxel: number;
+  boundingBoxTopLeft: Vector3;
+  boundingBoxPosition: Vector3;
+};
+
+const exportStatisticsToCSV = (
+  segmentInformation: Array<SegmentInfo>,
+  tracingId: string,
+  groupId: number,
```

Comment:
Is this the group id for which the user selected the export? If so, I'd rename it to `groupIdToExport` or something like that.